### PR TITLE
Hide TabBarBottom on keyboardDidShow

### LINKED ISSUE #618: TabBarBottom should hide when Keyboard is activated
Currently, the TabBarBottom is placed above the keyboard when you are typing. 
Sometimes it is displayed, sometimes it is not even properly rendered, and a white block is displayed.
Expected behavior would be that the Bar would disappear when the Keyboard becomes active.

Wondering if there is an easy way to implement this in the current version of react-navigation?

### PR DESCRIPTION
**Pull request by RalfNieuwenhuizen**
fixes #618

### CODE REVIEW

File: src/views/TabView/TabBarBottom.js [hunk 1]
```diff
@@ -57,6 +62,31 @@ export default class TabBarBottom extends PureComponent<DefaultProps, Props, voi
 
   props: Props;
 
+	keyboardDidShowListener: React.EmitterSubscription;
+	keyboardDidHideListener: React.EmitterSubscription;
+
+  state = {
```
Comment (matthamil) [inline src/views/TabView/TabBarBottom.js]: nit: Can you match the indentation of the source code? The indentation here is off.

**Comment by ericvicenti** [comment]
Does this apply for both iOS and Android? How have you tested this fix?

I think this approach is quite risky, so I think we should be very careful or else we will introduce bad behaviors

**Comment by RalfNieuwenhuizen** [comment]
If we can rely on the Keyboard component from react-native, this works on both platforms.

**Comment by grabbou** [comment]
Closing as this PR has pending review for a month.

Feel free to re-open or start a new one when you got time. My idea would be to try creating a higher order component that abstracts the keyboard logic and reuse it accordingly.

**Pull request closed without merging**